src/app/features/test-records/components/vehicle-header/vehicle-header.component.ts

```diff
@@ -0,0 +1,38 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { TestTypesTaxonomy } from '@api/test-types';
+import { TestResultModel } from '@models/test-results/test-result.model';
+import { TestType } from '@models/test-types/test-type.model';
+import { TechRecordModel } from '@models/vehicle-tech-record.model';
+import { TestTypesService } from '@services/test-types/test-types.service';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'app-vehicle-header',
+  templateUrl: './vehicle-header.component.html',
+  styleUrls: ['./vehicle-header.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class VehicleHeaderComponent {
+  @Input() isEditing = false;
+  @Input() testResult?: TestResultModel;
+  @Input() techRecord?: TechRecordModel | null | undefined;
+  @Input() testNumber?: string;
+
+  constructor(private testTypesService: TestTypesService) {}
+
+  get vehicleType(): string {
+    return this.techRecord?.vehicleType ?? '';
+  }
+
+  get test(): TestType | undefined {
```

Comment:
Same comment about using `this.testRecord.testType[0].testNumber` here, wouldn't need to pass the testNumber down to the component

Reply:
Don't want to keep making the assumption that there is 100% an index `0`. So although roundabout, its safer to find the test by the number we have in the url.